luacompiler/src/bytecode/mod.rs

```diff
@@ -0,0 +1,140 @@
+pub mod instructions;
+
+use bincode::{ serialize, deserialize };
+use std::vec::Vec;
+use std::fs::File;
+use std::fmt;
+use std::io::{ Result, Write };
```

Comment:
This is "overwriting" the normal `Result` struct with `io::Result` which is a bit confusing. I'd change this to `use std::io::{self, Write}` and refer to `io::Result` in the rest of the module.